server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/JamesMailSpooler.java

```diff
@@ -185,7 +172,7 @@ public void dispose() {
 
     @Override
     public int getThreadCount() {
```

Comment:
Should we rename the method in the parent interface MailSpoolerMBean ?

Reply:
it's part of the public API, it's why I didn't changed it for now